packages/account/Readme.md

```diff
@@ -0,0 +1,133 @@
+yarn add @biconomy/core-types @biconomy/account @biconomy/bundler @biconomy/paymaster
+```
+
+Building and sending userOperation is the main offering of any tool kit built for ERC4337. A prime example of such a toolkit is the Biconomy account package. Crafted with the needs of developers in mind, this package provides seamless integration of the crucial features associated with ERC-4337. It simplifies the process of building and sending UserOperations, thereby streamlining the development and management of decentralised applications (dApps).
+
+The Biconomy account package does this by providing a set of methods that make it easy for developers to create UserOperations. This, coupled with the sophisticated backend of the Biconomy platform, ensures efficient and reliable transmission of these operations on EVM networks.
+
+### Bundler
+
+In the context of  (ERC4337), the concept of a bundler plays a central role in the infrastructure. This concept is integral to the operation of account abstraction across any network that utilizes the Ethereum Virtual Machine (EVM). 
+
+Bundler infrastructure is designed and implemented in accordance with standardised specifications. This standardisation across all bundlers offers a significant advantage, particularly when it comes to interoperability with various tools and services, such as the Biconomy SDK.
+
+## configuration
+| Key                | Description |
+| -------------------| ------------- |
+| bundlerUrl         | Represent ERC4337 spec implemented bundler url. you can get one from biconomy dashboard. Alternatively you can supply any of your preferred|
+| chainId            | This represents the network your smart wallet transactions will be conducted on. Take a look following Link for supported chain id's |
+| entryPointAddress  | Since entrypoint can have different addresses you can call getSupportedEntryPoints() on bundler instance for supported addresses list|
+
+```typescript
+// This is how you create bundler instance in your dapp's
+import { IBundler, Bundler } from '@biconomy/bundler'
+import { ChainId } from "@biconomy/core-types";
+
+const bundler: IBundler = new Bundler({
+    bundlerUrl: '',      
+    chainId: ChainId.POLYGON_MAINNET,
+    entryPointAddress: '',
+  })
+```
+
+### Paymaster
+
+ERC4337, Account abstraction, introduces the concept of Paymasters. These specialised entities play a pivotal role in revolutionising the traditional gas payment system in EVM transactions. Paymasters, acting as third-party intermediaries, possess the capability to sponsor gas fees for an account, provided specific predefined conditions are satisfied.
+
+**paymasterUrl** you can get this value from biconomy dashboard.
+
+```typescript
+// This is how you create paymaster instance in your dapp's
+import { IPaymaster, BiconomyPaymaster } from '@biconomy/paymaster'
+
+  const paymaster: IPaymaster = new BiconomyPaymaster({
+    paymasterUrl: '' // you can get this value from biconomy dashboard.
```

Comment:
nit: we should add link to the biconomy dashboard whenever we mention to get a value from it

Reply:
resolved